rust/src/utils.rs

```diff
@@ -163,13 +163,18 @@ impl Deserialize for TransactionUnspentOutput {
 }
 
 #[wasm_bindgen]
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
```

Comment:
just FYI we're getting rid of these Vec-wrapper types outside of the wasm bindings in general since they were only needed due to `wasm_bindgen` restrictions. Although this struct would still need to exist, we'll just be moving it to the wasm bindings crate.